resource/server/init.lua

```diff
@@ -44,18 +44,22 @@ local function checkForUpdate()
 end
 
 local function determineOS()
-    local system = nil
-
-    local unix = os.getenv('HOME')
-    local windows = os.getenv('HOMEPATH')
-
-    if unix then system = 'unix' end
-    if windows then system = 'windows' end
-
-    -- this guy probably has some custom ENV var set...
-    if unix and windows then error('Couldn\'t identify the OS unambiguously.') end
+    local osType = GetConvar("os_type", "")
```

Comment:
Can you refer to some documentation or otherwise prove that this is actually working?